.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to Contribution Toolbox
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+# Contributor License Agreement (CLA)
+A CLA is a document that specifies how a project is allowed to use your
+contribution; they are commonly used in many open source projects.
+
+**_All_ contributions to _all_ projects hosted by [FINOS](https://www.finos.org/)
+must be made with a
+[Foundation CLA](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/83034172/Contribute)
+in place, and there are [additional legal requirements](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Legal+Requirements)
+that must also be met.**
+
+_NOTE:_ Commits and pull requests to FINOS repositories will only be accepted from those contributors with an active, executed Individual Contributor License Agreement (ICLA) with FINOS OR who are covered under an existing and active Corporate Contribution License Agreement (CCLA) executed with FINOS. Commits from individuals not covered under an ICLA or CCLA will be flagged and blocked by the FINOS Clabot tool. Please note that some CCLAs require individuals/employees to be explicitly named on the CCLA.
+
+As a result, PRs submitted to the Contrib Toolbox project cannot be accepted until you have a CLA in place with the Foundation.
+
+*Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
+
+# Contributing Issues
+
+## Prerequisites
+
+* [ ] Have you [searched for duplicates](https://github.com/finos-fdx/contrib-toolbox/issues?utf8=%E2%9C%93&q=)?  A simple search for exception error messages or a summary of the unexpected behaviour should suffice.
+* [ ] Are you running the latest version?
+* [ ] Are you sure this is a bug or missing capability?
+
+## Raising an Issue
+* Create your issue [here](https://github.com/finos-fdx/contrib-toolbox/issues/new).
+* New issues contain two templates in the description: bug report and enhancement request. Please pick the most appropriate for your issue, **then delete the other**.
```

Comment:
Do we have issue templates?

Reply:
We do, on https://github.com/finos/project-blueprint , but not on this repo (as we never had relevant external contributions yet).